### ISSUE TITLE
[Bug]: Butterfly won't get updated 

### ISSUE THREAD

**Issue by CodeDoctorDE** [vedantmgoyal2009/winget-pkgs-automation#218]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

https://github.com/LinwoodCloud/Butterfly/releases/tag/v1.0.0 I published this yesterday but I don't find a pr: https://github.com/microsoft/winget-pkgs/pulls?q=is%3Apr+butterfly

**Comment by vedantmgoyal2009** [comment]
Automation has been down for the last 12 hours or so. It will be detected in the next run after the issue is fixed.

**Comment by CodeDoctorDE** [comment]
Oh okay, I only was surprised because the last version was published so fast

**Comment by CodeDoctorDE** [comment]
It seems like the ruby version isn't up to date

**Comment by vedantmgoyal2009** [comment]
Yes, if you know anything about it, please tell me. I tried several things locally, but nothing seems to work.

**Comment by vedantmgoyal2009** [comment]
Thanks for opening the PR. I was trying to refactor the `gem` command to install a particular version of the gem.

> https://github.com/LinwoodCloud/Butterfly/releases/tag/v1.0.0 I published this yesterday but I don't find a pr: https://github.com/microsoft/winget-pkgs/pulls?q=is%3Apr+butterfly

Sorry, I went through the repository and saw that it was skipped because the current logic that detects if a PR for the update does already exist, mismatched the version (1.0.0 = 1.0.0-rc.6) and continued further without making a PR.

I have updated the package manually, but don't worry, I'm also developing a GitHub Action that any developer can use to update the package more instantly than the current automation does. Since Microsoft is more in favor of developers themself publishing the manifests for their package (however, it does not forbid to use of any other solution), it will help in doing so and then any developer will not have to rely on any other automation and its downtime won't affect the updation of the package. This automation will then be used to update non-GitHub packages and the packages that developers don't want to automate package themselves.

**Comment by CodeDoctorDE** [comment]
Oh it heards nice.
I'm currently using the https://semver.org/
Yes, the current approche is a bit confusing

**Comment by vedantmgoyal2009** [comment]
Yes, I know you are using SemVer, but GitHub API doesn't parse it to

**Comment by CodeDoctorDE** [comment]
https://github.com/microsoft/winget-pkgs/pulls?q=butterfly I don't find the pr currently

**Comment by CodeDoctorDE** [comment]
can you please look up why there is no pr for this version in the community repo?

**Comment by vedantmgoyal2009** [comment]
Sorry for the inconvenience. I created a PR manually. I did run the script, but it failed at the automatic PR creation step.

**Comment by CodeDoctorDE** [comment]
I don't know what to do but winget does not detect 1.0.0 as the latest version
````
Name                                               ID                                Version       Verfügbar     Quelle
-----------------------------------------------------------------------------------------------------------------------
Butterfly                                          LinwoodCloud.Butterfly            1.0.0         1.0.0-rc.6    winget
````

Do you know what I can do?

**Comment by CodeDoctorDE** [comment]
Oh thanks for your work!
In the future, when I released maybe 1.1.0-beta.1, ..., can we change that only the stable versions will get released?
It's a bit complicated to mark them as pre-releases when my internal tags are marked as it too. It will get very confusing.
